### ISSUE TITLE
Errors in FireFox

### ISSUE THREAD

**Issue by akpraha** [foundernest/namecase#1]
While testing a react app on various browsers, after adding this library, FireFox (latest version 75) reports an error during page load:

SyntaxError: invalid regexp group

**Comment by jesobreira** [comment]
Same in Safari (desktop and mobile)

SyntaxError: Invalid regular expression: invalid group specifier name

```javascript
a.capitalizeFirstLetters = function (t) {
    return (t = t.toLowerCase()).replace(/(?<=[\s,.:;"'(-]|^)./g, function () { // <-- error on this line
      for (var t = arguments.length, e = new Array(t), a = 0; a < t; a++) {
        e[a] = arguments[a];
      }

      return e[0].toUpperCase();
    });
  }
```

**Comment by daniseijo** [comment]
It looks related with this problem: https://bugzilla.mozilla.org/show_bug.cgi?id=1425763

I'll try to look into it.

If you manage to fix the bug, be sure to make a PR with the changes!

**Comment by jesobreira** [comment]
It requires proper testing but I think we could rewrite the regexp without the look-behind. 

```javascript
.replace(/([\s,.:;"'(-]|^)(.)/g, (...matches) => matches[0].toUpperCase())
```

I spotted more usage of look-behind in the code that should also be rewritten. Does above's regexp complies to the one in my previous comment?

**Comment by SimonPrague** [comment]
I've created a fix that works without this look-behind regexps.

This is the PR: https://github.com/foundernest/namecase/pull/3

**Comment by SimonPrague** [comment]
Surprisingly this bug only showed up in Safari and Firefox was ok. But nevertheless this is a great library and I'd love to use is as soon as this PR is merged! Thanks!

**Comment by daniseijo** [comment]
It seems like Firefox fixed this error in some version between v75 and the current one. However, I downloaded version 75 and the fix in #3 also fix the firefox v75 issue. Thank you again!
